lively.collab/comments/commentMorph.js

```diff
@@ -277,16 +277,16 @@ export class CommentMorph extends Morph {
   }
 
   hideCommentIndicator () {
```

Comment:
should we rename this method?

Reply:
Is it not semantically correct?

Reply:
I think it is at least not obvious, that this method actually abandons the Indicator. I would expect that I can retrieve something that I hid later on, but this is not possible here. I can get a `CommentIndicator` for the Morph again, but not the one I hide here, because it is abandoned. 
It is a small difference, but I would be in favor of `abandonCommentIndicator()`. Or do I miss something here?